mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -2723,3 +2765,66 @@ resource "google_container_node_pool" "np" {
 }
 
 
+func testAccContainerNodePool_concurrentCreate(cluster, np1, np2 string) string {
+	return fmt.Sprintf(`
+provider "google" {
```

Comment:
You should be able to omit this block- I think it may have been necessary for you to test locally and opt-in to the new behaviour?

Reply:
Yep, that was a remnant from my local testing. Removed.